README.md

```diff
@@ -33,8 +34,10 @@ Feel free to contribute or post issues to the [GitHub repository](https://github
 
 ## Note for Developers
 
-- You will need a Last.fm API key and secret to run it: http://www.last.fm/api
-- Place your key and secret in `api/config.js`. The file `config.sample.js` is provided as a template.
+-   You will need a Last.fm API key and secret to run it: http://www.last.fm/api
+-   Place your key and secret in `api/config.js`. The file `config.sample.js` is provided as a template.
+-   To get started, install Node ^10.16.3 with npm ^6.12.0, then run `npm i && npm watch`
```

Comment:
Almost works but I needed to run npm run watch instead. Does this also work for you?

Reply:
Good catch on the typo. I corrected it.